packages/hash/frontend/src/pages/[account-slug]/entities/[entity-uuid].page/entity-editor/properties-section/property-table/use-row-data/generate-property-rows-from-entity/generate-property-row-recursively.ts

```diff
@@ -63,26 +71,27 @@ export const generatePropertyRowRecursively = (
 
   const required = !!requiredPropertyTypes?.includes(propertyTypeBaseUri);
 
-  let value;
+  const value =
+    get(entity.properties, propertyKeyChain) ?? (isArray ? [] : undefined);
+
   const children: PropertyRow[] = [];
 
-  if (isUnknownObject(properties)) {
-    value = properties[propertyTypeBaseUri];
+  const firstOneOf = propertyType.oneOf[0];
```

Comment:
because we assume if there is a nested property object, it holds one `oneOf` and that's a ref to the nested property (so we generate the rows, in future we can extend it to be dynamic but that's how it's initially coded, and it's out-of-scope from this PR. This behavior is not introduced in this PR)